docs/src/modules/components/TopLayoutBlog.js

```diff
@@ -231,7 +231,56 @@ function TopLayoutBlog(props) {
         disableAlternateLocale
         card={card}
         type="article"
-      />
+      >
+        <meta name="author" content={headers.authors.map((key) => authors[key].name).join(', ')} />
+        <meta property="article:published_time" content={headers.date} />
+        <script
```

Comment:
Moved to the `<head>` to fix the duplicate presence: https://validator.schema.org/#url=https%3A%2F%2Fmui.com%2Fblog%2Fmui-x-v6-alpha-zero%2F

<img width="695" alt="Screenshot 2022-10-31 at 02 24 50" src="https://user-images.githubusercontent.com/3165635/198913196-94b8ffdb-82d2-4e47-a9b4-0b32004b939e.png">